eks/cluster-loader/clusterloader2/addon.go

```diff
@@ -99,5 +99,17 @@ func (c *ClusterLoader) buildConfigMapData() (map[string]string, error) {
 }
 
 func (c *ClusterLoader) buildArgs() string {
-	return fmt.Sprintf("[%s]", strings.Join(c.Config.Spec.ClusterLoader.TestParams, ","))
+	return fmt.Sprintf("[%s]", strings.Join(append(c.Config.Spec.ClusterLoader.TestParams, "--report-dir=/usr/share/cluster-loader"), ","))
+}
+
+func (c *ClusterLoader) buildS3Command() string {
```

Comment:
I think it might be a little easier to read the script if it were a go template and you filled in the blanks? Would that be possible? Not a huge deal if it's too big of a change.

Reply:
I was in the habit of making it fully configurable in go, but it does make more sense to put most of it in the gotemplate. Will do.